ddtrace/span.py

```diff
@@ -165,6 +170,19 @@ def _ignore_exception(self, exc):
         else:
             self._ignored_exceptions.append(exc)
 
+    @removed_property(
+        message="Accessing a tracer from a span is no longer supported. Span.tracer will be removed.",
```

Comment:
This message is appended by the removed property annotation: "Reading the 'Span.tracer' property is deprecated and will be removed in version '1.0.0". 

Should we remove the message argument and just keep the default warning message?

Right now we're generating this message:

` Reading the 'Span.tracer' property is deprecated and will be removed in version '1.0.0': Accessing a tracer from a span is no longer supported. Span.tracer will be removed. `

Reply:
I removed the message argument from `@removed_property`.

Current deprecation warning: `Reading the 'Span.tracer' property is deprecated and will be removed in version '1.0.0'`